tests/common.go

```diff
@@ -259,6 +259,7 @@ const (
 )
 
 var pxRuntimeOpts string
+var UserContextBackupLocation string = ""
```

Comment:
You dont have to give empty string here. In golang by default every string is empty